rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  def initialize(_date, _book, _person)
+    @date = date
+    @person = person
+    person.rentals << self
+
+    @book = book
+    book.rentals << self
+  end
+  attr_accessor :date, :book, :person
```

Comment:
- [ ] I would recommend you move this declaration to the top for better code structure.